src/slotmap.js

```diff
@@ -24,6 +24,19 @@ const {
     storageLayout: { storage, types },
 } = require('../out/HtsSystemContract.sol/HtsSystemContract.json');
 
+let ledgerId = '0x00';
+/**
+ * Sets the `ledgerId` used when retrieving fungible and non-fungible tokens.
+ * The `ledgerId` depends on the `chainId` of the remote network it is forking from.
+ *
+ * @param {number} chainId
+ */
+const setLedgerId = chainId => {
```

Comment:
let's export this one from `index.d.ts`. Also we can move the doc comments there, so consumers can take advantage of the js docs.

nit: convert to `function`.